mbed_lstools/main.py

```diff
@@ -174,6 +181,9 @@ def get_mbedls_version():
     mbeds.debug(__name__, "mbed-ls ver. " + get_mbedls_version())
     mbeds.debug(__name__, "host: " +  str((mbed_lstools_os_info())))
 
+    if not opts.skip_retarget:
+        mbeds.retarget()
```

Comment:
I don't think this step belongs here, it's already handled in the lstools_base class: https://github.com/ARMmbed/mbed-ls/blob/a1aec616f93e0c2eb68889d38e8053918975b8bc/mbed_lstools/lstools_base.py#L77

Reply:
Heh, You're right.